overlord/devicestate/devicestate_install_mode_test.go

```diff
@@ -3897,8 +3897,8 @@ func (s *installStepSuite) TestDeviceManagerInstallFinishRunthrough(c *C) {
 	st.Lock()
 
 	c.Check(chg.IsReady(), Equals, true)
-	// TODO: update once the change actually does something
```

Comment:
we should keep a TODO, as this is one of the test to expand to unit test the handler. As the current failure is a lack of setup